modules/osdk-control-compat.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-working-bundle-images.adoc
+
+[id="osdk-control-compat_{context}"]
+= Controlling Operator compatibility with {product-title} versions
+
+[IMPORTANT]
+====
+Kubernetes periodically deprecates certain APIs that are later removed in subsequent releases. As a result, your Operator project will be unable to use any removed APIs starting with the version of {product-title} that uses whichever Kubernetes version removed the API.
+
+As an Operator author, it is strongly recommended that you review the link:https://kubernetes.io/docs/reference/using-api/deprecation-guide/[Deprecated API Migration Guide] in Kubernetes documentation and ensure that your projects stay updated to avoid using deprecated and removed APIs, ideally before the release of a future version of {product-title} that would make the Operator incompatible.
+====
+
+When an API is removed from an {product-title} version, Operators running on that cluster version that are still using any removed APIs will no longer work properly. As an Operator author, you should plan to update your Operators to accommodate API deprecation and removal to avoid interruptions for users of your Operator.
+
+[TIP]
+====
+You can check the event alerts of your Operators running on {product-title} 4.8 and later to find whether there are any warnings about APIs currently in use. The following alerts fire when they detect an API in use that will be removed in the next release:
+
+`APIRemovedInNextReleaseInUse`::
+APIs that will be removed in the next OpenShift Container Platform release.
+
+`APIRemovedInNextEUSReleaseInUse`::
+APIs that will be removed in the next OpenShift Container Platform link:https://access.redhat.com/support/policy/updates/openshift#ocp4_phases[Extended Update Support (EUS)] release.
+====
+
+To prevent workflow issues, cluster administrators must install a version of your Operator in their cluster that is compatible with the next version of {product-title} before they upgrade their cluster from any previous version. While it is recommended that you update your Operators to no longer use deprecated or removed APIs, if you still need to publish your Operator bundles with removed APIs for continued use on earlier versions of {product-title}, ensure that the bundle is configured accordingly.
```

Comment:
To my inexperienced brain, the first sentence has a lot going on.  Does it change the meaning if you remove some words?

To prevent workflow issues, cluster administrators must install a version of your Operator that is compatible with the next version of {product-title} before they upgrade.

Or, maybe "Before cluster administrators upgrade to the next version of {product-title}, they must install a version of your Operator that is compatible with the next version."

Or, it might be more complicated than my brain wants to allow.

Reply:
Went with:

```
If a cluster administrator has installed your Operator, before they upgrade to the next version of {product-title}, they must install a version of your Operator that is compatible with that next cluster version.
```